src/core/tests/TestAptitudePackageManager.py

```diff
@@ -24,10 +24,18 @@ class TestAptitudePackageManager(unittest.TestCase):
     def setUp(self):
         self.runtime = RuntimeCompositor(ArgumentComposer().get_composed_arguments(), True, Constants.APT)
         self.container = self.runtime.container
+        self.valid_auto_os_update_settings = 'APT::Periodic::Update-Package-Lists "1";' + \
```

Comment:
Updating package lists shouldn't install any packages. It may be beneficial to leave it alone so that reporting within the OS outside of cloud orchestration remains accurate (like listing number of packages available for update when you login).

Reply:
As discussed, disabling updating packages as well